test/test.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('chai').assert;
+var index = require('../index.js');
+var app = require('../app.js');
+var request = require('request');
+const https = require('https');
```

Comment:
[Y]ES6!

Also, go to Australia. You have to wait a few months before you pop back up and make me sad. That's the way @benesch does it.